sympy/physics/mechanics/_actuator.py

```diff
@@ -703,7 +703,7 @@ def at_pin_joint(
 
         >>> actuator = TorqueActuator.at_pin_joint(torque, pin_joint)
         >>> actuator
-        TorqueActuator(T, axis=N.z, target_frame=N, reaction_frame=A)
+        TorqueActuator(T, axis=N.z, target_frame=A, reaction_frame=N)
```

Comment:
You could add a note on the direction, e.g. that a positive torque will result in a positive displacement of the joint coordinate or that the torque is applied on the child body with a reaction torque on the parent.

Reply:
Sure then, I will add this change and push it into the repo

Reply:
Could you rephrase it to be part of the previous paragraph, something like:
```
... of the ``TorqueActcuator`` class. It should be noted that a positive torque will cause a positive displacement of the joint coordinate.
```